packages/core/database/tsconfig.build.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist",
-    "noEmit": false
```

Comment:
is it removed because rollup ignores it regardless of tsconfig.json having noEmit: true ?

Reply:
yes: https://github.com/strapi/strapi/blob/aa136be2a7ee226962a561a70fe0a0ca1e73d708/packages/utils/pack-up/src/node/core/tsconfig.ts#L44-L51